backend/resources/task_detail.py

```diff
@@ -140,5 +140,9 @@ def get(self):
             return {"message": "An error occured to get the task detail"}, 500
         if task_detail:
             task_list = [task.json() for task in task_detail]
+            for task in task_list:
+                if not task["finish_time"]: 
+                    task["finish_time"] = 0
+            task_list.sort(key=lambda x: x.get("finish_time"))
```

Comment:
can we sort by start_time instead and use the database's sort instead of sorting this manually?